app/models/manageiq/providers/vmware/infra_manager/provision_via_pxe/state_machine.rb

```diff
@@ -30,10 +30,7 @@ def boot_from_network
       _log.info("#{destination_type} [#{dest_name}] is not yet ready to boot, will retry")
       requeue_phase
     else
-      # Temporarily set the database raw_power_state in case the refresh has not come along yet.
-      destination.update_attributes(:raw_power_state => "wait_for_launch")
-
-      signal :poll_destination_powered_off_in_vmdb
+      signal :poll_destination_powered_on_in_provider
```

Comment:
@bdunne Noticed we are calling `destination.start` here but using `start_with_cache_reset ` when starting the VM from `autostart_destination`.  What do you think about fixing that up in this PR as well?